### DISCUSSION TITLE
Does the Ciphey team take decryption challenges?

### DISCUSSION THREAD

**Discussion by SamMatzko** [bee-san/Ciphey#722] [Q&A] [answered]
If I come up with a encryption algorithm, and share it with you, and then give you a message encrypted using that algorithm but without giving you the key, would you try to crack it (by coming up with a code algorithm that does so)? I'm simply curious whether my encryption is breakable, and I'm not a good code-breaker; this isn't a feature request or anything. Do you like or dislike decryption challenges?

**Comment by Jayy001** [comment, marked as answer]
Sure, join the discord and post it in #coded-message

**Reply by SamMatzko** [reply]
Okay! I'm not on Discord, and I haven't come up with an algorithm yet, so you probably won't see me there for a while. But I'm glad to know that it's possible. Thanks for the feedback!

**Reply by Jayy001** [reply]
No problem, looking forward to seeing it!
